cargo-miri/bin.rs

```diff
@@ -398,20 +398,22 @@ fn setup(subcommand: MiriCommand) {
     if !dir.exists() {
         fs::create_dir_all(&dir).unwrap();
     }
-    // The interesting bit: Xargo.toml
-    File::create(dir.join("Xargo.toml"))
-        .unwrap()
-        .write_all(
-            br#"
+    let mut xargo_toml = File::create(dir.join("Xargo.toml")).unwrap();
+    if std::env::var_os("MIRI_NO_STD").is_none() {
+        // The interesting bit: Xargo.toml
+        xargo_toml
+            .write_all(
+                br#"
 [dependencies.std]
 default_features = false
 # We support unwinding, so enable that panic runtime.
 features = ["panic_unwind", "backtrace"]
 
 [dependencies.test]
 "#,
-        )
-        .unwrap();
+            )
+            .unwrap();
+    }
```

Comment:
So, we just don't write a `Xargo.toml` for the other targets and that works?

Reply:
We create an empty one. Not having one didn't work

Reply:
Right, but I am still surprised an empty one does anything. :shrug: I guess if the test passes that's all I can ask for.^^

Reply:
The xargo docs say that by default only libcore is built